### PR TITLE
Add ASWF Digitial Assets license v1.0

### PR DESCRIPTION
**Pull request by jmertic**
Signed-off-by: John Mertic <jmertic@linuxfoundation.org>

### CODE REVIEW

File: digital_assets/aswf_digital_assets_license.txt [hunk 1]
```diff
@@ -0,0 +1,17 @@
+ASWF Digital Assets License v1.0
+
+License for <Asset Name, e.g. “Moana Island Scene”> (the “Asset Name”).
```
Comment (swinslow) [inline digital_assets/aswf_digital_assets_license.txt]: I'd remove "Moana Island Scene" -- so changing this line to just say:

`License for <Asset Name> (the “Asset Name”).`

File: digital_assets/README.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+## License Attribution
+
+Assets should include language within it's metadata indicating the use of this license and pointing to the full license text. Example below:
+
+```
+ASWF Digital Assets License v1.0 (available at https://aswf.io/licenses/aswf_digital_assets_license_v1.0.txt)
+```
+
+You can also use the SPDX Short Form identifer as shown below:
```
Comment (swinslow) [inline digital_assets/README.md]: I'd add a bit more detail here -- to say something like:

You can also use a license identifier in the custom SPDX "LicenseRef-" format, such as the following. Please note that this license is _not_ currently on the [SPDX License List](https://spdx.org/licenses), so this identifier uses the "LicenseRef-" prefix and will not be independently recognized by scanning tools.

File: digital_assets/README.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+- research
+- software and hardware development
+- performance benchmarking (including publication of benchmark results and permitting reproducibility of the benchmark results by third parties)
+- software and hardware product demonstrations.
+
+Version 1.0 of this license was approved for use by the Academy Software Foundation Governing Board on 2021-07-29. This license is available to used by both ASWF and non-ASWF projects.
+
+## License Attribution
+
+Assets should include language within it's metadata indicating the use of this license and pointing to the full license text. Example below:
```
Comment (swinslow) [inline digital_assets/README.md]: s/it's/their

File: digital_assets/README.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+```
+ASWF Digital Assets License v1.0 (available at https://aswf.io/licenses/aswf_digital_assets_license_v1.0.txt)
```
Comment (swinslow) [inline digital_assets/README.md]: I'd add a bit more of a prefix, to say: "Released under the ASWF Digital..."

File: digital_assets/aswf_digital_assets_license.txt (hunk 1 above)
Comment (swinslow) [reply digital_assets/aswf_digital_assets_license.txt]: Would also make the same change in the version that gets posted at aswf.io

File: digital_assets/README.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+- education
+- training
+- research
+- software and hardware development
+- performance benchmarking (including publication of benchmark results and permitting reproducibility of the benchmark results by third parties)
+- software and hardware product demonstrations.
+
+Version 1.0 of this license was approved for use by the Academy Software Foundation Governing Board on 2021-07-29. This license is available to used by both ASWF and non-ASWF projects.
+
+## License Attribution
```
Comment (swinslow) [inline digital_assets/README.md]: s/Attribution/Notices

File: digital_assets/README.md [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+This folder contains a license for use with for assets intended to be used solely for any of the following purposes:
+
+- education
+- training
+- research
+- software and hardware development
+- performance benchmarking (including publication of benchmark results and permitting reproducibility of the benchmark results by third parties)
+- software and hardware product demonstrations.
+
+Version 1.0 of this license was approved for use by the Academy Software Foundation Governing Board on 2021-07-29. This license is available to used by both ASWF and non-ASWF projects.
```
Comment (swinslow) [inline digital_assets/README.md]: "available to _be_ used"

File: digital_assets/README.md [hunk 7]
```diff
@@ -0,0 +1,26 @@
+# ASWF Digital Assets License 
+
+This folder contains a license for use with for assets intended to be used solely for any of the following purposes:
```
Comment (swinslow) [inline digital_assets/README.md]: for use with ~for~ assets

File: digital_assets/README.md [hunk 8]
```diff
@@ -0,0 +1,26 @@
...
+- training
+- research
+- software and hardware development
+- performance benchmarking (including publication of benchmark results and permitting reproducibility of the benchmark results by third parties)
+- software and hardware product demonstrations.
+
+Version 1.0 of this license was approved for use by the Academy Software Foundation Governing Board on 2021-07-29. This license is available to used by both ASWF and non-ASWF projects.
+
+## License Attribution
+
```
Comment (swinslow) [inline digital_assets/README.md:15]: I'd include the following near the top of this section:

When releasing assets under this license, be sure to replace the angle bracketed sections in the header of the template (`<Asset Name>`, `<Year>`, `<Asset Owner>`) with the applicable information for your asset. You may wish to consult with your own legal counsel if you have questions about the specific details to include.

**Review by swinslow — CHANGES_REQUESTED** [review]
Thanks @jmertic! Looks good, I had a handful of tweaks to suggest -- see below. Let me know if you have any questions.

File: digital_assets/aswf_digital_assets_license.txt (hunk 1 above)
Comment (jmertic) [reply digital_assets/aswf_digital_assets_license.txt]: Done - note I'm thinking the URI for aswf.io will just be a forward to this version. You think that will be ok?

**Comment by jmertic** [comment]
Thanks Steve! Made the changes suggested.

Made one comment above on the license URI - thinking of having a URI under aswf.io that forward to here. Not sure if that is ok.

**Comment by swinslow** [comment]
Thanks!

On the license URI... hmm. My one concern is that redirecting to a GitHub address might lead to it pointing to the wrong version, if/when the license is updated in the future. Or you might have to redirect to the GitHub URI for a specific commit, which could get confusing too.

I think if you point to the GitHub URI for the version 1.0 of the license, that's likely fine as long as you can make sure that the file isn't edited in the future, and the repo isn't reorganized or the path changed, etc. That could get complicated but could be fine if you think that's manageable.

**Comment by jmertic** [comment]
Good point - I was thinking for new versions I'd just add a whole new file and URI which might make this easier. But I likely am also overthinking things :-)

**Review by swinslow — APPROVED** [review]

**Pull request merged**